src/Package.php

```diff
@@ -39,6 +46,13 @@ public static function init() {
 
 		// Avoid double initialization when the feature plugin is in use.
 		if ( defined( 'WC_ADMIN_VERSION_NUMBER' ) ) {
+			$update_version = new WC_Admin_Notes_Deactivate_Plugin();
+			if ( version_compare( WC_ADMIN_VERSION_NUMBER, self::VERSION, '<' ) ) {
+				$update_version::add_note();
+			} else {
+				$update_version::delete_note();
```

Comment:
If the `WC_ADMIN_VERSION_NUMBER` is not defined, that means the plugin is either inactive or not installed, so the notice should not be displayed. 
However, in my testing with WC Admin 0.23.3, the code never hits `$update_version::delete_note();`.